From 2316f2b96f362ed2d3e03161dccda272c75948b9 Mon Sep 17 00:00:00 2001
From: Xinyu Chen <xinyu.chen@freescale.com>
Date: Sun, 6 Jan 2013 14:24:56 +0800
Subject: [PATCH 140/602] ENGR00238982 imx6: include Google GMS package config
 file

The file is under device/google/gapps/.
gapps.mk is used to install all of the files
in GMS package except .apk.
gapps_config.mk is used to override the
BOARD_SYSTEMIMAGE_PARTITION_SIZE configure, which
requires 450MB for GMS pacakge installed.

Signed-off-by: Xinyu Chen <xinyu.chen@freescale.com>
---
 imx6/BoardConfigCommon.mk | 2 ++
 imx6/imx6.mk              | 2 ++
 2 files changed, 4 insertions(+)

diff --git a/imx6/BoardConfigCommon.mk b/imx6/BoardConfigCommon.mk
index ff3b7b0..81325a9 100644
--- a/imx6/BoardConfigCommon.mk
+++ b/imx6/BoardConfigCommon.mk
@@ -59,3 +59,5 @@ BOARD_RECOVERY_PARTITION_SIZE = 10M
 
 BOARD_SYSTEMIMAGE_PARTITION_SIZE := 293601280
 BOARD_FLASH_BLOCK_SIZE := 4096
+
+-include device/google/gapps/gapps_config.mk
diff --git a/imx6/imx6.mk b/imx6/imx6.mk
index 7d0d50d..d4244bb 100644
--- a/imx6/imx6.mk
+++ b/imx6/imx6.mk
@@ -322,3 +322,5 @@ PRODUCT_DEFAULT_DEV_CERTIFICATE := \
 
 # include a google recommand heap config file.
 include frameworks/native/build/tablet-7in-hdpi-1024-dalvik-heap.mk
+
+-include device/google/gapps/gapps.mk
-- 
1.8.0

